### DISCUSSION TITLE
Use same webcam on host and remote

### DISCUSSION THREAD

**Discussion by Raniz85** [FreeRDP/FreeRDP#9300] [General]
If I redirect the webcam USB device it disappears from my host computer.

However, I do video calls on both devices so I would like to be able to use the same webcam on both the host and remote computer - not at the same time though.

I can do this by reconnecting with or without the USB redirection, but if possible I'd rather have a solution that doesn't require me to do this.

**Comment by hardening** [comment]
if you redirect the webcam with the USB redirector, FreeRDP takes the ownership of the USB device and so you will not see it anymore locally. The solution would be webcam redirection , but the client part of that channel is not implemented yet in FreeRDP.

**Reply by akallabeth** [reply]
https://github.com/FreeRDP/FreeRDP/issues/4971
